### DISCUSSION TITLE
Allow Test notifications to also set a severity

### DISCUSSION THREAD

**Discussion by adean-consensys** [grafana/grafana#71675] [Alerting]
When testing the notifications from a contact point, I would like the ability to specify a severity, because our incident management system (for example, OpsGenie) has specific rules around who receives a P1 versus a P3, and we would like to test this flow using notification tests.

Feature ask is to add a dropdown here to include severity (Default is P3)

![image](https://github.com/grafana/grafana/assets/83668448/82b6335c-eb6e-4fa6-b6ec-710719c73a05)

**Comment by flhelfer** [comment]
Hi @adean-consensys,

You can specify a label with a given serverity. For example, with an integration to Opsgenie, you can use the label "og_priority". It seems that it is a dedicated variable used by opsgenie.

![image](https://github.com/grafana/grafana/assets/8560200/abb932f9-253e-4746-84d3-cd9b23003300)

Valid values are P1/P2/P3/P4/P5.

For the label to be able to overwrite the priority, you also need to check the "Override prioty" property in you contact point under "Optional OpsGenie settings".

On Opsgenie side, make sure the "Priority" field use the variable "{{priority}}".

![image](https://github.com/grafana/grafana/assets/8560200/0e2719cc-b987-45c3-a488-ed8bdac143db)

I agree that a dropdown list would be easier, but I guess that "priority" variable is different between all the integrationsprovider. Therefore, this mean that the "test contact point" pop-up wouldn't be generic.

Here is a [link](https://community.atlassian.com/t5/Opsgenie-questions/Sending-priority-from-Grafana/qaq-p/1259302) that helped me.

Hope my answer helped you with your tests!

Florian

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82617. Feel free to continue the discussion around this there. Thank you!
